docs/source/core_developers_guide.rst

```diff
@@ -0,0 +1,12 @@
+
+**********************
+Core Developer's Guide
+**********************
+
+.. toctree::
+   :maxdepth: 2
+
+   core_developers_guide/sawtooth_vagrant
+   core_developers_guide/intkey_workload.rst
```

Comment:
Looks like core_developers_guide/sawtooth_vagrant path should be core_developers_guide/sawtooth_vagrant.rst, since there is no "sawtooth_vagrant" directory in core_developers_guide.